tests/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/Fixtures/Even.php';
```

Comment:
Haven't written PHP in a long time but I remember that if you set up `require-dev` ( which you have), you don't have to manually require files except `vendor/autoload.php`

Reply:
Yeah, I almost forgot that. I only to do is loading the `vendor/autoload.php` and don't load that files you mention manually because of namespace definition.